kpops/components/streams_bootstrap/streams/model.py

```diff
@@ -224,28 +230,122 @@ class PersistenceConfig(BaseModel):
     )
 
 
+class PrometheusConfig(CamelCaseConfigModel, DescConfigModel):
+    """Prometheus JMX exporter configuration.
+
+    :param jmx: The prometheus JMX exporter configuration.
+
+    """
+
+    class PrometheusJMXConfig(CamelCaseConfigModel, DescConfigModel):
+        """Prometheus JMX exporter configuration.
+
+        :param enabled: Whether to install Prometheus JMX Exporter as a sidecar container and expose JMX metrics to Prometheus.
+        :param image: Docker Image for Prometheus JMX Exporter container.
+        :param image_tag: Docker Image Tag for Prometheus JMX Exporter container.
+        :param image_pull_policy: Docker Image Pull Policy for Prometheus JMX Exporter container.
+        :param port: JMX Exporter Port which exposes metrics in Prometheus format for scraping.
+        :param resources: JMX Exporter resources configuration.
+        """
+
+        enabled: bool = Field(
+            default=True,
+            description=describe_attr("enabled", __doc__),
+        )
+        image: str = Field(
+            default="solsson/kafka-prometheus-jmx-exporter@sha256",
```

Comment:
is that correct? the suffix looks a little weird

Reply:
Extreamly weird but it is the default value: https://github.com/bakdata/streams-bootstrap/blob/master/charts/streams-app/values.yaml#L154